apex-squid/src/test/java/org/fundacionjala/enforce/sonarqube/apex/parser/grammar/ApexGrammarDmlOperationsTest.java

```diff
@@ -23,43 +23,38 @@
  */
 package org.fundacionjala.enforce.sonarqube.apex.parser.grammar;
 
+import org.fundacionjala.enforce.sonarqube.apex.parser.ApexRuleTest;
 import org.junit.Before;
 import org.junit.Test;
-
-import org.fundacionjala.enforce.sonarqube.apex.parser.ApexRuleTest;
-
+import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.ApexGrammarRuleKey.DML_OPERATIONS;
 import static org.sonar.sslr.tests.Assertions.assertThat;
 
-import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.ApexGrammarRuleKey.CONSTRUCTOR_DECLARATION;
-
-public class ApexGrammarConstructorDeclaratorTest extends ApexRuleTest {
+public class ApexGrammarDmlOperationsTest extends ApexRuleTest {
 
     @Before
     public void init() {
-        setRootRule(CONSTRUCTOR_DECLARATION);
-    }
-
-    @Test
-    public void testConstructor() {
-        assertThat(parser)
-                .matches("public Book(){}")
-                .matches("public Account(){int myVariable;}")
-                .notMatches("public Table()")
-                .notMatches("public int save(){");
+        setRootRule(DML_OPERATIONS);
     }
 
     @Test
-    public void testConstructorWithParameter() {
+    public void rulesDmlOperations() {
         assertThat(parser)
-                .matches("public Book(int id){}")
-                .matches("public Table(int items[]){}")
-                .notMatches("public int update(intleft,intright)");
+                .matches("insert primaryExpression")
```

Comment:
this would be a valid scenario too

insert primaryExpression

insert primaryExpression;